### ISSUE TITLE
Filter Options for Custom Fields - Improvement needed

### ISSUE THREAD

**Issue by coolcat-creations** [joomla/joomla-cms#29052]
### Steps to reproduce the issue
Custom Fields have own filter options, when they are not changed, means when their setting is "Plugin-Settings" a WYSIWYG Editor filters out all html. Which does not make sense at all. The editor field should respect the set up Joomla Textfilters. It's ok to have deeper settings to, but the default should be the Joomla Textfilters...

### Expected result
If you create for examplel an editor custom field the field should have the Joomla Textfilter as default

### Actual result
If you create for example an editor custom field the field strips out all html by default

### System information (as much as possible)
Joomla 3.9.18

**Comment by SharkyKZ** [comment]
https://github.com/joomla/joomla-cms/pull/28721 should partially solve this.

**Comment by coolcat-creations** [comment]
Not really, it solves that tiny mce is using textfilters but the custom fields don't use the tinymce settings.
